lib/runner.rb

```diff
@@ -12,4 +12,5 @@
   game_teams: game_teams_path
 }
 
-stat_tracker = StatTracker.from_csv(locations)
\ No newline at end of file
+require 'pry'; binding.pry
```

Comment:
Do we need to remove the pry before merging?

Reply:
done!

Reply:
Thanks Ashley!